manifests/params.pp

```diff
@@ -4,6 +4,59 @@
 # @api private
 #
 class vault::params {
+  $user               = 'vault'
```

Comment:
@bastelfreak sorry for that. I wasn't aware. It's been ages since I did some puppet development actually... Never got on the voxpupuli train, and now I am not really doing real puppet module development.

Nevertheless, to double check, you are not only talking about just the $user variable? But about all other static ones too. Right?

Reply:
hey, no problem. yes, please move all static data. we also have some review guidelines at https://voxpupuli.org/docs/reviewing_pr/